translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/ProjectManager/activities/ActivityProjectManager.java

```diff
@@ -199,6 +199,7 @@ private void initializeViews() {
 
         hideProjectsIfEmpty(mNumProjects);
         if (mNumProjects > 0) {
+            calculateProjectsProgress();
```

Comment:
I don't want to put this here; there's already too much work going on to start up this activity that we need to make sure that things that won't change very often don't happen all the time.

progress is only "wrong" when the filesystem and database were out of sync, correct?
Then let's add a resultcode to the resync to return whether or not anything changed, and only on change do we calculate all the project progress.